src/OrbitGl/ScopeTree.h

```diff
@@ -109,6 +112,22 @@ ScopeNode<ScopeT>* ScopeTree<ScopeT>::CreateNode(ScopeT* scope) {
   return &new_node;
 }
 
+template <typename ScopeT>
+ScopeT* ScopeTree<ScopeT>::GetRight(uint32_t depth, uint64_t time) const {
+  auto nodes_at_depth = GetOrderedNodesByDepth().at(depth);
```

Comment:
I think we should CHECK that the map contains the passed in `depth` and not rely on the exception.

Reply:
Do you still want this now that we're using a depth that comes from a ScopeNode?

Reply:
No I think it's fine, thanks.